lib/cocoapods/installer/analyzer.rb

```diff
@@ -74,117 +83,111 @@ def analyze(allow_fetches = true)
         return @result if @result
         validate_podfile!
         validate_lockfile_version!
-        @result = AnalysisResult.new
-        @result.podfile_dependency_cache = @podfile_dependency_cache
         if installation_options.integrate_targets?
-          @result.target_inspections = inspect_targets_to_integrate
+          target_inspections = inspect_targets_to_integrate
```

Comment:
this method `analyze` is now more of a pipeline, outputs of one method are inputs to another, like `pods_project_generator`

Reply:
No actual logical changes.